docs/providers/route53.rst

```diff
@@ -5,6 +5,8 @@ route53
 
     * ``private_zone`` Indicates what kind of hosted zone to use. if true, use only private zones. if false, use only public zones
 
+    * ``zone_id`` The aws hostedzone id to use; e.g. 'a1b2zabcdefghi'
```

Comment:
` ./utils/generate_provider_doc.py` alters that, I've decided to include it into this MR rather than creating a separate one.

Reply:
Don't bother, `generate_provider_doc` is called as part of the releasing process, so it will be run ultimately.